### PR TITLE
added training section with links to Scylla University

### PR DESCRIPTION
**Pull request by guy9**
Added training section to Readme file

### CODE REVIEW

**Comment by guy9** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -250,6 +250,16 @@ There are various ways to bind application level data structures to CQL statemen
 * [gocassa](https://github.com/hailocab/gocassa) is an external project that layers on top of gocql to provide convenient query building and data binding.
 * [gocqltable](https://github.com/kristoiv/gocqltable) provides an ORM-style convenience layer to make CRUD operations with gocql easier.
 
+Training
```
Comment (dahankzter) [inline README.md]: Maybe call it something else? Perhaps "Resources" or "More knowledge and examples" or something?

File: README.md [hunk 2]
```diff
@@ -250,6 +250,16 @@ There are various ways to bind application level data structures to CQL statemen
...
+--------
+
+[Scylla University](https://university.scylladb.com/) includes training material and online courses which will help you become a Scylla NoSQL database expert.
```
Comment (dahankzter) [inline README.md]: Perhaps we can remove this generic Scylla link and keep only the Go and driver related links?

File: README.md [hunk 3]
```diff
@@ -250,6 +250,16 @@ There are various ways to bind application level data structures to CQL statemen
...
 * [gocqltable](https://github.com/kristoiv/gocqltable) provides an ORM-style convenience layer to make CRUD operations with gocql easier.
 
+Training
+--------
+
+[Scylla University](https://university.scylladb.com/) includes training material and online courses which will help you become a Scylla NoSQL database expert. 
+The course [Using Scylla Drivers](https://university.scylladb.com/courses/using-scylla-drivers/) explains how to use drivers in different languages to interact with a Scylla cluster.
+The lesson, [Golang and Scylla Part 1](https://university.scylladb.com/courses/using-scylla-drivers/lessons/golang-and-scylla-part-1/) includes a sample application that, using the GoCQL driver, interacts with a three-node Scylla cluster. 
+It connects to a Scylla cluster, displays the contents of a  table, inserts and deletes data, and shows the contents of the table after each action.
+Courses in [Scylla University](https://university.scylladb.com/) cover a variety of topics with an emphasis on installing Scylla, Scylla in production and other NoSQL database training concepts – including Scylla DBA training.
```
Comment (dahankzter) [inline README.md]: Same for this generic Scylla link. It's the same as the one above and the users will find it from the driver and Go specific links.

Comment (guy9) [reply README.md]: Done

**Comment by guy9** [comment]
@dahankzter thanks for the input I made the changes, please review

**Comment by guy9** [comment]
ping

**Comment by dahankzter** [comment]
Looks good now, thanks!

**Pull request merged**